### ISSUE TITLE
Question about multiple value recipients

### ISSUE THREAD

**Issue by hollingsworthd** [Podcastindex-org/podcast-namespace#653]
Great work on Podcasting 2.0.

tl;dr: in the spec, specifying a required single crypto recipient and optional multiple recipients seems necessary

I am curious how podcasting apps are expected to implement multiple recipients on iOS. From my understanding Apple would be more likely to approve an app if the crypto is handled by a separate wallet app. Can I specify multiple recipients in a "lighting:" URI to launch a user's preferred wallet app or are there any other options? I've tried and it doesn't work. It does work for single recipients, e.g. to the Muun wallet.

It might be nice to have a property to specify a single recipient address along with suggested splits and addresses. Otherwise afaik I have to load the wallet N times for each recipient, let user manually click on each address, or have the app just choose one of the recipients. For the time being I'll choose the recipient with the largest share and assume they will redistribute it. Other options are too complex for the user and harder for the recipients to untangle (assuming the user does not follow through on donating to all recipients manually at the recommended split).

Perhaps smart contracts that Podcasting 2.0 maintains could help here, if the recipient is a contract and if that's possible. I think it's possible on Hedera which uses Solidity contracts and eth. There would need to be some API or UI which allows groups of recipients to set up these contracts.
